libs/dh/market-participant/actors/feature-actors/src/lib/drawer/dh-actor-drawer.component.ts

```diff
@@ -28,13 +28,32 @@ import {
   WattDescriptionListItemComponent,
 } from '@energinet-datahub/watt/description-list';
 import { WATT_CARD } from '@energinet-datahub/watt/card';
-import { GetActorByIdDocument } from '@energinet-datahub/dh/shared/domain/graphql';
+import {
+  ActorAuditLogType,
+  ContactCategory,
+  GetActorByIdDocument,
+  GetAuditLogByActorIdDocument,
+} from '@energinet-datahub/dh/shared/domain/graphql';
 
 import { DhActorExtended } from '../dh-actor';
 import { DhActorStatusBadgeComponent } from '../status-badge/dh-actor-status-badge.component';
 import { DhActorsEditActorModalComponent } from '../edit/dh-actors-edit-actor-modal.component';
 import { WattButtonComponent } from '@energinet-datahub/watt/button';
 import { DhPermissionRequiredDirective } from '@energinet-datahub/dh/shared/feature-authorization';
+import { VaterStackComponent } from '@energinet-datahub/watt/vater';
+import { WattSpinnerComponent } from '@energinet-datahub/watt/spinner';
+import { WattEmptyStateComponent } from '@energinet-datahub/watt/empty-state';
+import { WATT_TABLE, WattTableColumnDef, WattTableDataSource } from '@energinet-datahub/watt/table';
+import { WattDatePipe } from '@energinet-datahub/watt/date';
+
+type ActorAuditLogEntry = {
+  changeType: ActorAuditLogType;
+  currentValue: string;
+  previousValue: string;
+  category: ContactCategory;
+  identity: string;
+  timestamp: Date;
+};
```

Comment:
suggestion: instead of repeating the same type returned by the query, it can be taken from the query itself. Similar to `DhOrganization` in `libs\dh\market-participant\actors\feature-organizations\src\lib\dh-organization.ts`.

Reply:
Fixed